engine/src/test/scala/cromwell/engine/backend/WorkflowDescriptorSpec.scala

```diff
@@ -53,7 +53,8 @@ trait WorkflowDescriptorBuilder {
     implicit val timeout = akka.util.Timeout(awaitTimeout)
     implicit val ec = actorSystem.dispatcher
 
-    val actor = actorSystem.actorOf(MaterializeWorkflowDescriptorActor.props(), "MaterializeWorkflowDescriptorActor-" + id.id)
+    val serviceRegistryIgnorer = actorSystem.actorOf(DevNullActor.props)
```

Comment:
[`Props.empty`](https://github.com/akka/akka/blob/v2.3.12/akka-actor/src/main/scala/akka/actor/Props.scala#L49) does the same as your `DevNullActor.props`:

```scala
val serviceRegistryIgnorer = actorSystem.actorOf(Props.empty)
```